src/main/java/org/junit/internal/builders/JUnit4Builder.java

```diff
@@ -1,12 +1,12 @@
 package org.junit.internal.builders;
 
 import org.junit.runner.Runner;
-import org.junit.runners.BlockJUnit4ClassRunner;
+import org.junit.runners.JUnit4;
 import org.junit.runners.model.RunnerBuilder;
 
 public class JUnit4Builder extends RunnerBuilder {
     @Override
     public Runner runnerForClass(Class<?> testClass) throws Throwable {
-        return new BlockJUnit4ClassRunner(testClass);
+        return new JUnit4(testClass);
     }
-}
\ No newline at end of file
+}
```

Comment:
It is a separate commit. See https://github.com/junit-team/junit4/pull/1449/commits/11822c4fd1d2388079ed6a2eadd61ed7b3d44035

I have this commit in this pull since there's no reason for `ParentRunner.createTestClass()` to be called for JUnit4-style tests that aren't annotated with `RunWith`.